vendor/engines/secure_rooms/app/views/secure_rooms/facility_occupancies/index.html.haml

```diff
@@ -0,0 +1,5 @@
+= content_for :h1 do
+  =h current_facility
+
+= content_for :tabnav do
+  = render :partial => 'secure_rooms/shared/tabnav_occupancy'
```

Comment:
We're trying to switch our standard to use 1.9 hash syntax, double quotes, and not using `partial` and `locals` in `render` where posssible. So `render "secure_rooms/shared/tabnav_occupancy"`